Bender.yml

```diff
@@ -11,21 +11,21 @@ package:
     - "Yvan Tortorella <yvan.tortorella@unibo.it>"
 
 dependencies:
-  register_interface: { git: https://github.com/pulp-platform/register_interface.git,   version: 0.4.2                                }
+  register_interface: { git: https://github.com/pulp-platform/register_interface.git,   version: 0.4.3                                }
   axi:                { git: https://github.com/pulp-platform/axi.git,                  version: 0.39.1                               }
-  cheshire:           { git: https://github.com/pulp-platform/cheshire.git,             rev: 0c95210cf242c384fafe3019e84b8974c3ff1e92 } # branch: aottaviano/carfield
+  cheshire:           { git: https://github.com/pulp-platform/cheshire.git,             rev: d6274a52dab68d12765b64d670d528774eb36c2c } # branch: idma-rebase
```

Comment:
Is this the branch used in Astral? So astral changes on top of carfield changes on top of cheshire **current** main?